README.md

```diff
@@ -1,10 +1,32 @@
 # Programming with Python
+
 [![GitHub release](https://img.shields.io/github/release/swcarpentry/python-novice-inflammation.svg)](https://github.com/swcarpentry/python-novice-inflammation/releases)
 [![Create a Slack Account with us](https://img.shields.io/badge/Create_Slack_Account-The_Carpentries-071159.svg)](https://swc-slack-invite.herokuapp.com/)
 [![Slack Status](https://img.shields.io/badge/Slack_Channel-swc--py--inflammation-E01563.svg)](https://swcarpentry.slack.com/messages/C9Y0L6MF0)
 
 An introduction to Python for non-programmers using inflammation data.
 
+Current maintainers:
+
+* [Trevor Bekolay][trevor_bekolay]
+* [Maxim Belkin][maxim_belkin]
+* [Anne Fouilloux][anne_fouilloux]
+* [Valentina Staneva][valentina_staneva]
+* [Mike Trizna][mike_trizna]
+* [Greg Wilson][greg_wilson], creator of [Software Carpentry][swc_history]
+
+The goal of this lesson is to teach novice programmers to write modular code to
```

Comment:
Let's not say what the goal of the lesson is. Let's just say that this lesson teaches <blah>.